### PR TITLE
Adding assesment 2

### PR DESCRIPTION
**Pull request by GuillaumeChamp**
may need correction

### CODE REVIEW

File: Assignment2/GuillaumeChamp-GuillaumeChamptoussel.rdf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	:includes :sensor029,
+	:computer101.
+:computer101
+	:hasOwner User101.
+:User10A
+	:hasName pedro.
+:Sensor029
+	:hasMeasurement Measurement8401.
+:Measurement8401
+	:has Temperature29.
```
Comment (ocorcho) [inline Assignment2/GuillaumeChamp-GuillaumeChamptoussel.rdf:14]: This one does not look good.

File: Assignment2/GuillaumeChamp-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "@context :{
+    "my":"https://github.com/GuillaumeChamp",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "fullName": "rdfs:label",
+    "hometown": "rdfs:label",
```
Comment (ocorcho) [inline Assignment2/GuillaumeChamp-fromJSONtoJSON-LD.jsonld:6]: Not all properties should be transformed to rdfs:label.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/GuillaumeChamp-fromJSONtoJSON-LD.jsonld (hunk 2 above)
Comment (GuillaumeChamp) [reply Assignment2/GuillaumeChamp-fromJSONtoJSON-LD.jsonld:6]: Can we not specify the type of a property or is it better to define them using a non-adapted one ?

**Comment by GuillaumeChamp** [comment]
error fixed + assesment 3

**Pull request merged**